navbar.js

```diff
@@ -0,0 +1,9 @@
+// eslint-disable-next-line no-unused-vars
```

Comment:
- [ ] I'll kindly advise you to always fix issues raised by Linters. Linters are tools that enable us to detect and flag programming, and stylistic bugs/errors. Instead of disabling them, let's fix these issues. Let's do the same in line 6 below.